docs_src/pipelinerun.md

```diff
@@ -43,28 +43,56 @@ Use docker-run.sh to run the pipeline
 
 ### option 1 to run object detection, object classification, OCR, and barcode recognition with simuated camera:
```

Comment:
typo: should be simulated

Reply:
i think there is conflict with the main branch so need to resolve that first: this should be fixed after that

Reply:
yeah....I need to rebase